First commit of Project Blueprint Docusaurus white label template

mcleo-d opened the pull request:
This is the first PR of the FINOS Project Blueprint Docusaurus white label template for FINOS projects to refer to when creating project related wikis and other local documentation.

mcleo-d commented:
Hi @maoo - Can you review and accept this PR if appropriate as it will form the foundations of further FINOS work in the pipeline for @grizzwolf

maoo commented on website/build/Project Blueprint/blog/2016/03/11/blog-post.html:
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html><html lang="en"><head><meta charSet="utf-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><title>Blog Title · FINOS Project Blueprint</title><meta name="viewport" content="width=device-width"/><meta name="generator" content="Docusaurus"/><meta name="description" content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque elementum dignissim ultricies. Fu ...
```
`website/build` must be:
1. removed from Git
2. added to `.gitignore`

maoo commented:
Wow, 18244 files changed :-)

Please make sure to never (ever) commit build files (ie `website/build`) and - most importantly - the `node_modules` folder.

Also, given that we don't have a preview feature to see the changes, it would be good to attach some screenshots of what been done.

Last, I don't understand why we would want to use Docusaurus to serve blog entries for our projects; this would lead to have one single blog "feed" for each project (which, on average, I'd expect to be almost empty); from a marketing PoV, I believe it would be much more useful to consolidate entries into www.finos.org/blog . So, my suggestion would be to disable blogs from the website blueprint.

mcleo-d commented:
Yikes, sorry @maoo - Well done catching the `.gitignore` not being added to the project.

Let me go back and review why this file was not added as part of the original commit 👍

mcleo-d commented:
Hi @maoo, 

I can confirm I have done the following according to your feedback ...

- Added the `.gitignore` which was excluded from the original commit. 
- Removed the `build` and `node_modules` folders and content from the repo.
- Provided the following screenshots of the build.

I can also confirm the blog is a feature of Docusaurus that's disabled in the `siteConfig.js` where the current config is outlined below ...

```
headerLinks: [
  {doc: 'finos-home', label: 'Docs'},   
  {doc: 'service-catalog', label: 'Use Cases'},
  {doc: 'finos-developer-identity', label: 'Get Involved'},
  {doc: 'service-catalog', label: 'Why Project Blueprint'},
  {blog: false}
]
```

mcleo-d commented:
![Screenshot 2020-02-04 at 13 59 36](https://user-images.githubusercontent.com/6029572/73751982-0b453300-4758-11ea-8052-509ccfd6c1c8.png)

mcleo-d commented:
![Screenshot 2020-02-04 at 13 59 48](https://user-images.githubusercontent.com/6029572/73752022-18fab880-4758-11ea-92f2-434fe9e325e9.png)

mcleo-d commented:
@maoo - I have now brought the repo in line with the `.gitignore` configuration so committed files have now been removed from repo 👍

maoo commented:
Thanks for the update, looks much better now! I still see content in `website/blog` though, which I'd remove to keep things simple.

mcleo-d commented:
After a conversation with @maoo, the Docusaurus blog functionality has been removed from FINOS Project Blueprint.

maoo commented:
LGTM! TY @mcleo-d !!

maoo commented:
@brooklynrob - the idea is to keep things simple and avoid adding items that are not part of the current website blueprint we used so far (fdc3, fo, alloy).

Also, and Ill let @toshaellison comment on it, I think it would be good to push ppl to contribute to finos.org/blog , instead of managing multiple semi-empty blog instances.
